packages/frontend/pages/lobby.tsx

```diff
@@ -14,138 +18,325 @@ import Chat, { dateGenObj } from '../components/lobby/chat';
 import GamesMenu from '../components/lobby/gamesMenu/gamesMenu';
 import Announcements from '../components/lobby/announcements';
 
-interface IProps {
+interface IStateProps {
   theme: ThemeOptions;
   mobileView: MobileView;
+  windowDimensions: WindowDimensions;
 }
 
-const Lobby = ({ theme, mobileView }: IProps): ReactElement => {
+const Lobby = ({
+  mobileView,
+  theme,
+  windowDimensions,
+}: IStateProps): ReactElement => (
+  <>
+    <title>Lobby</title>
+    {mobileView ? (
+      <LobbyMobile theme={theme} windowDimensions={windowDimensions} />
+    ) : (
+      <LobbyDesktop />
```

Comment:
Is it possible to have LobbyMobile and LobbyDesktop in separate files? :)

Reply:
Yes should I put them in components/lobby?